### ISSUE TITLE
microcks 1.10.0 chart conflicts with the mongodb bitnami chart.

### ISSUE THREAD

**Issue by rquinio1A** [microcks/microcks#1263]
### Describe the bug

When both microcks and bitnami mongodb chart are included in the same umbrella chart, there's an Helm error:

```
Error: template: umbrella/charts/mongodb/templates/standalone/dep-sts.yaml:93:12: executing "umbrella/charts/mongodb/templates/standalone/dep-sts.yaml" at <include "mongodb.initContainer.prepareLogDir" .>: error calling include: template: umbrella/charts/mongodb/templates/_helpers.tpl:276:12: executing "mongodb.initContainer.prepareLogDir" at <include "mongodb.image" .>: error calling include: template: umbrella/charts/microcks/templates/_helpers.tpl:103:52: executing "mongodb.image" at <.Values.mongodb.image>: nil pointer evaluating interface {}.image
```

Both charts happen to define the "mongodb.image" named template.

### How to Reproduce?

Create an umbrella with:

```yaml
dependencies:
- name: mongodb
  version: 15.6.16
  repository: alias:bitnami
- name: microcks
  version: 1.10.0
  repository: alias:microcks
```

**Comment by lbroudoux** [comment]
Hi @rquinio1A,
Is it really related to `1.10.0`? The `mongodb.image` property used to be there for a while.

**Comment by rquinio1A** [comment]
I got the issue when upgrading microcks 1.9.1 -> 1.10.0, I think it became visible after the type change from mongodb.image="" to mongodb.image={}, it must have worked before by chance, depending on which template was winning the merge.

On bitnami side: https://github.com/bitnami/charts/blob/e3c893177b27a510e933db97d4f57f1d80d1bfc7/bitnami/mongodb/templates/_helpers.tpl#L52

**Comment by lbroudoux** [comment]
Oh ok got it! Thanks a lot to the PR BTW!

**Comment by lbroudoux** [comment]
Fixed in PR #1264
